stable/openvpn/README.md

```diff
@@ -132,7 +132,10 @@ Certificates can be found in openvpn pod in the following files:
  `/etc/openvpn/certs/pki/ca.crt`
  `/etc/openvpn/certs/pki/issued/server.crt`
  `/etc/openvpn/certs/pki/dh.pem`
+ `/etc/openvpn/certs/pki/crl.pem`
 
 And optionally (see openvpn.taKey setting):
 
  `/etc/openvpn/certs/pki/ta.key`
+
+Note: using mounted secret makes creation of new client certificates impossible inside openvpn pod, since easyrca needs to write in certs directory, which is read-only.
```

Comment:
Nitpick: typo here, `easyrca` should be `easyrsa`

Reply:
fixed